### ISSUE TITLE
Is it possible to upload CSV with translation?

### ISSUE THREAD

**Issue by tpimh** [opennota/opennota#5]
I want to upload CSV file with translation.
```
First column | Second column
----------------------------
sourcetext1  | translation1
sourcetext2  | translation2
sourcetext3  | translation3
```
Is it possible? Is it possible to download the translation as CSV?
Comma is used as field separator and strings may contain commas. Maybe it's a better idea to use TSV format?

**Comment by opennota** [comment]
No, it's not possible. Are you using this code (I would be surprised) or just want to upload to the [test server](http://opennota.duckdns.org:9000/)? In the first case, I recommend you [uisky/notabenoid](https://github.com/uisky/notabenoid). This code is crappy and isn't supported anymore. In the second case, you do not want to rely on the availability of the test server. It can be brought down at any time.

**Comment by tpimh** [comment]
I just did upload my translation [here](http://opennota.duckdns.org:9000/#/book/193).
I want to move my translation process from github to some collaborative editing website. Now it is hosted [here](https://github.com/tpimh/angband-r/tree/master/edit.tsv).

Your project was recommended [here](https://www.linux.org.ru/forum/development/11752485?cid=11752533).

**Comment by opennota** [comment]
You shouldn't rely on opennota.duckdns.org

**Comment by tpimh** [comment]
Is notabenoid source hosted somewhere for public access? Does notabenoid support import/export CSV or TSV?

**Comment by opennota** [comment]
> Is notabenoid source hosted somewhere for public access?

http://notabenoid.org is invitation only. I'm not aware of other deployments yet.

> Does notabenoid support import/export CSV or TSV?

No, AFAIK.

**Comment by opennota** [comment]
@tpimh I implemented basic CSV import in notabenoid [here](https://github.com/notabenoid/notabenoid/tree/csv).
